src/stan/services/arguments/arg_variational.hpp

```diff
@@ -31,6 +31,7 @@ namespace stan {
                                  "of ELBO (objective function)",
                                  100));
         _subarguments.push_back(new arg_variational_eta());
```

Comment:
The convention for member variables is to use final underscores, not initial ones.  Initial ones will run into trouble with the C++ spec if they get capitalized.